src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const man = people.filter(
+    person => person.sex === 'm'
+      && (!century || century
+        === Math.ceil(person.died / 100))
+  );
+
+  const manAge = man.map(person => person.died
```

Comment:
you can shorten these two lines of code, just count the difference in the reduce method